gamestate.py

```diff
@@ -0,0 +1,24 @@
+from playingcards import Card
+from playingcards import Deck
+import random
+
+class Player:
+    def __init__(self, name):
+        #self.hand = hand[]
+        self.name = name
+
+deck = Deck()
+deck.show()
+random.shuffle(deck.cards)
+print("now shuffled\n")
+deck.show()
+print(deck.cards[2].s)
+print("{} of {}".format(deck.cards[51].s, deck.cards[51].n))
+print ("some bullshit")
+deck.cards[51].show()
```

Comment:
I think this is the bullshit you're referring to because it's really bad practice to have global code running at the top level of a file. Everything should be in a function, so it's clear that the only time it gets run is when you actually call that function.